src/App.js

```diff
@@ -22,18 +28,44 @@ export default class App extends Component {
         featuredBeast : beast
       })
         this.showModal();
+      }
+
       
-    }
-  render() {
+        
+      handleHorn =(selection) =>{
+        let updatedHornArray
+        
+      if (selection === "1"){
+          updatedHornArray = Horneddata.filter(i=>i.horns===1);
+          this.setState ({allHorns : updatedHornArray})
+           }
+          
+          else if (selection === "2"){
+            updatedHornArray = Horneddata.filter(i=>i.horns===2);
+            this.setState ({allHorns : updatedHornArray})
+
+         }
+          else if (selection === "3"){
+              updatedHornArray = Horneddata.filter(i=>i.horns===3);
+              this.setState ({allHorns : updatedHornArray})
+          
+              }
+          else if (selection === "morethan3"){
+                updatedHornArray = Horneddata.filter(i=>i);
+                this.setState ({allHorns : updatedHornArray})
+                
+          }
```

Comment:
Off to a good start!

Feedback here: Grab a TA to see if they can help you with filter. You have the right idea here but you can do this using filter() once 👍